BridgeSDK/BridgeAPI/SBBBridgeNetworkManager.h

```diff
@@ -36,8 +36,30 @@
 
 @end
 
+
+/**
+ * If the app delegate conforms to this protocol then the methods included will be called as appropriate.
+ */
+@protocol SSBBridgeAppDelegate <UIApplicationDelegate>
```

Comment:
Two things: (1) typo, the "namespace" prefix for this framework is SBB not SSB, and (2) should this maybe be declared in BridgeSDK.h or possibly SBBAuthManager.h so AppCore and other Bridge clients don't have to reference (or even know about) SBBBridgeNetworkManager?